src/CashinReportGenerator/Program.cs

```diff
@@ -0,0 +1,413 @@
+            ServiceProvider = collection.BuildServiceProvider();
+            //Lykke.Job.EthereumCore.Config.RegisterDependency.RegisterJobs(collection);
+            //var web3 = ServiceProvider.GetService<Web3>();
+            //web3.Eth.GetBalance.SendRequestAsync("");
+            // web3.Eth.Transactions.SendTransaction.SendRequestAsync(new Nethereum.RPC.Eth.DTOs.TransactionInput()
+            //{
+            //    
+            //}).Result;
+            //var key = EthECKey.GenerateKey().GetPrivateKeyAsBytes();
+            //var stringKey = Encoding.Unicode.GetString(key);
+            ServiceProvider = collection.BuildServiceProvider();
+
+            //string coinAdapterAddress = settings.CoinAdapterAddress;
+            //string balancesInfoConnString = settings.BalancesInfoConnString;
+            //string bitCoinQueueConnectionString = settings.BitCoinQueueConnectionString;
+            //string clientPersonalInfoConnString = settings.ClientPersonalInfoConnString;
+            //string ethAssetId = settings.EthAssetId;
+
+            //var log = ServiceProvider.GetService<ILog>();
+            //var bcnRepositoryReader = new BcnClientCredentialsRepository(
+            //        new AzureTableStorage<BcnCredentialsRecordEntity>(clientPersonalInfoConnString,
+            //            "BcnClientCredentials", log));
+            //var pendingActions = new EthererumPendingActionsRepository(
+            //        new AzureTableStorage<EthererumPendingActionEntity>(bitCoinQueueConnectionString,
+            //            "EthererumPendingActions", log));
+            //var privateWalletsReader = new PrivateWalletsRepository(
+            //        new AzureTableStorage<PrivateWalletEntity>(clientPersonalInfoConnString,
+            //            "PrivateWallets", log));
+            //var wallets = new WalletsRepository(new AzureTableStorage<WalletEntity>(balancesInfoConnString, "Accounts", log));
+           // var assetContractService = ServiceProvider.GetService<AssetContractService>();
+            var samuraiApi = ServiceProvider.GetService<ITransactionRepository>();
+            var ethPrecision = BigInteger.Pow(10, 18);
+            string command = "-1";
+            Console.WriteLine("Type 0 - to make feesReport(Cashin) for address");
+            //Console.WriteLine("Type 1 - to make feesReport(Cashout) for address");
+            //Console.WriteLine("Type 1 - to make cashinReport");
+            //Console.WriteLine("Type 2 - to make balance report");
+            //Console.WriteLine("Type 3 - to fill pending actions for users");
+
+            Console.WriteLine("Type exit - to quit");
+
+            while (command != "exit")
+            {
+                command = Console.ReadLine();
+
+                switch (command)
+                {
+                    case "0":
+                    {
+                        GenerateCashinReportAsync(samuraiApi);
```

Comment:
"Type 0 - to make feesReport(Cashin) for address" not matches GenerateCashinReportAsync(samuraiApi);